CHANGES.md

```diff
@@ -1,3 +1,13 @@
+# 3.0.0 - 02 July 2025
+
```

Comment:
We should add in that we raised the minimum version to 3.10. A manual edit to the PR seems fine, no need to have the automation do it.